main.py

```diff
@@ -78,7 +78,7 @@ def signup():
     if 1 == 0:
```

Comment:
Shouldn't this use the check_if_user_exists_by_email function in db.py?

Reply:
Thanks! Optimized both methods and added check email method to main.